internal/broker/handlers.go

```diff
@@ -65,6 +65,11 @@ func (c *Conn) onSubscribe(mqttTopic []byte) *Error {
 		return ErrUnauthorized
 	}
 
+	// Keys which are supposed to be extended should not be used for publishing
```

Comment:
// Keys which are supposed to be extended should not be used for publishing
---->
// Keys which are supposed to be extended should not be used for subscribing